### ISSUE TITLE
Why does it look so bad?

### ISSUE THREAD

**Issue by h-lame** [lrug/lrug.org#17]
The design of lrug.org is mostly "a dev did it without much thought", we should apply some modern thinking and make it nicer.  Things to think about:

* mobile experience (currently no margins and horrid)
* no red dashed borders (so 2006)
* one of those cool webfonts ([this one](https://www.google.com/fonts/specimen/Mountains+of+Christmas) perhaps?)
* &c
* &c

**Comment by h-lame** [comment]
This issue, much like the milestone, really just a placeholder for some more broken down effort later.

**Comment by MikeRogers0** [comment]
I'd be super up for having a devs-do-design session & seeing if I can come up with a design feels more modern. If I was to take the current design & made it responsive (maybe also throw in some modern stack approaches), would that be a good start?

![Bob Ross](https://media.giphy.com/media/n5vlIXw4XBgGc/source.gif)

**Comment by h-lame** [comment]
Making what we have work better on smaller screens would definitely be a good thing.  And a fresher design in general also sounds good.  It's quite subjective though, so I wouldn't want you to put in loads of effort and have us be "nah".  Is there an obvious light-weight way of proposing a new design without actually putting in the effort to do it?

"modern stack approaches" definitely sounds interesting, but it really depends what you mean.  The site design right now is p. simple so we don't really need SCSS, let alone more complex build-chains that rely on the asset-pipeline or webpack or whatever.  That said, perhaps these toolchains would let us explore better design, so it's a chicken-and-egg problem really.

I guess if you were to come up with a new design that we all liked, and the price of implementing that is adopting a more complex toolchain that's probably worth it.  But maybe not for it's own sake.  Does that make sense?

**Comment by MikeRogers0** [comment]
@h-lame Make sense :) I think I'll start with making a small change, then that might open up the conversation to what we'd like to tweak next :)

**Comment by MikeRogers0** [comment]
Awesome, thank you for assisting in that :)

The next steps I'd like to look at are:

1. Remove the `normalize.css` from the repo, instead pull it in via Yarn. 
2. Find out what people want expect the LRUG website. E.g. Should it just tell people about meetups, or be a wider gateway into learning Ruby/Getting a Ruby job.

**Comment by h-lame** [comment]
> Remove the normalize.css from the repo, instead pull it in via Yarn.

`yarn` isn't a dependency of the project at the moment, can we avoid introducing it?  Are there other ways to pull in `normalize.css` if it's something that is updated frequently enough we need to keep it super fresh?   On it's own, pulling `normalise.css` from some other source doesn't feel like a compelling enough reason to add `yarn` and `node_modules`, what else that we want to do would this enable for us?

If we do need to start involving `node_modules` could we use `npm` instead of `yarn`?  My experience with `yarn` in rails projects is that it's pretty inflexible.  I also thought that `npm` had surpassed `yarn` in functionality (e.g. the things `yarn` added that `npm` didn't have at the time `yarn` was created, `npm` now has so `yarn` feels redundant).

> Find out what people want expect the LRUG website. E.g. Should it just tell people about meetups, or be a wider gateway into learning Ruby/Getting a Ruby job.

I'd be super interested in this.  I suspect some folk _might_ want that, but also, not want to have to spend any effort on it.  Be useful to ask people what they want from the site, but also how they would get involved to manage and maintain it.  Lets find out though!

**Comment by MikeRogers0** [comment]
`normalize.css`/`yarn`: I'll do some research into options. I'm keen to keep 3rd party files out of our repo, though we could always come back to it another time.

> Be useful to ask people what they want from the site, but also how they would get involved to manage and maintain it. Lets find out though!

Agreed, even if we can't change it massively it would be nice to know what ideas people have. Would it be possible to put out a Tweet saying: ~"We're keen to improve https://lrug.org - What would you change to help serve the local ruby community?"

Off the top of my head, I'm keen to work displaying local jobs with fewer clicks. I found http://lists.lrug.org/pipermail/chat-lrug.org/2020-September/thread.html which lists some but it feels disconnected from the main site.

**Comment by h-lame** [comment]
> Agreed, even if we can't change it massively it would be nice to know what ideas people have. Would it be possible to put out a Tweet saying: ~"We're keen to improve https://lrug.org - What would you change to help serve the local ruby community?"

Might be best to come up with a survey and get folk to fill that in.  We've used google forms for this in the past, would you be interested in coming up with something?

**Comment by MikeRogers0** [comment]
@h-lame Yeah, I can put together a survey :) I _should_ have some time next week, what is the best way to setup a google form? I'm guessing we'd want to do it via a LRUG owned google account?

**Comment by h-lame** [comment]
> @h-lame Yeah, I can put together a survey :) 

Cool, thanks!

> I'm guessing we'd want to do it via a LRUG owned google account?

If such a thing existed, perhaps.  In the past when we've needed things like this we've just done them from our personal accounts and shared access amongst the organisers.  So I think it'd be fine that way this time too (assuming you have a personal google account and were willing to share those details with us so we could collaborate).

**Comment by MikeRogers0** [comment]
I've started working on a survey, getting the questions right is a little tricky but so far I've come up with:

## LRUG - Website User Insights

We're looking to improve our current website to support existing users, while also helping new members find their feet.

To help us, we'd like to ask you a few questions about how you use the LRUG website, your responses will help us decide what we should focus our development efforts on.

Responses of this survey will be public.

1. How often do you visit the LRUG website?
    - Every Week
    - Once a month
    - A few times a year
    - I've never visited the LRUG website

2. What would be your primary purpose for visiting the LRUG website?
    - Find out about the next event
    - Looking for information about Ruby
    - Searching for local Ruby jobs
    - To find Rubyists from London
    - To listen to episodes of the LRUG Podcast
    - Read the mailing list
    - Other ( Please State )

3. Where do you normally go to find out about up coming LRUG events?
    - LRUG's Website
    - Twitter
    - Eventbrite
    - The mailing list
    - Oher (Please State)

4. Is there anything you think should be changed on the existing LRUG website?
( Open ended question, so a text response here )


5. Are there any user groups you consider to have a really brilliant website?
( Open ended question, so a text response here )

What do you think?

**Comment by h-lame** [comment]
That survey sounds good.  Might I suggest adding:

> to find links to videos, blog posts, slides, etc... for talks

to Q2?  Other than that, looks good.

**Comment by MikeRogers0** [comment]
Awesome. I've imported that into a Google Form https://docs.google.com/forms/d/e/1FAIpQLScgrBLpCv7tHvxCgfe2TuKAM4iL-JPbzEdJ_Unk7OFH_VNpuA/viewform?usp=sf_link

If you have an email address I can add you as a collaborator if you'd like? / How do we go about sharing the link publicly?

**Comment by h-lame** [comment]
You can add me as: `murray DOT steele AT gmail DOT com` - I'll then add the other LRUG folk that would be interested in the results.  I think you could email the list explaining you've been doing some work on the website recently and wanted to get some insight on what to work on next.  We'll then do a tweet with the url from the email.

**Comment by MikeRogers0** [comment]
I've added you & sent it to the mailing list :) 🤞

**Comment by MikeRogers0** [comment]
We started getting responses @h-lame :D Would we be able to share the link on Twitter / mention it at the next meetup?
